src/hooks/useMarketIndication.ts

```diff
@@ -0,0 +1,83 @@
+import { useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+import { useGetTokenPrice } from 'libs/queries';
+import { Token } from 'libs/tokens';
+import { useFiatCurrency } from './useFiatCurrency';
+import { OrderCreate } from 'components/strategies/create/useOrder';
+
+export type MarketPricePercentage = {
+  min: BigNumber;
+  max: BigNumber;
+  price: BigNumber;
+};
+
+export type UseMarketIndicationProps = {
+  base: Token | undefined;
+  quote: Token | undefined;
+  order: OrderCreate;
+  buy?: boolean;
+};
+
+export const useMarketIndication = ({
```

Comment:
lets discuss this hook, I dont think this is a hook that should be places in the global hooks folder. And I have a feeling we are over complicating things because this data is used only in create strategy, right?

I see the hook beeing used these two files within the same scope:

`src/components/strategies/create/BuySellBlock/LimitRangeSection.tsx`
`src/components/strategies/create/useCreateStrategy.ts`

Reply:
pls see the comment above - we need this hook for both edit & create strategy